src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,37 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+
+/**
+ * The Command class represents the command entered by user.
+ */
+public abstract class Command {
+    private final boolean isTerminator;
+
+    /**
+     * Initializes an instance of a Command.
+     *
+     * @param isTerminator Indicates whether the command is a terminator.
+     */
+    protected Command(Boolean isTerminator) {
+        this.isTerminator = isTerminator;
+    }
+
+    public boolean isTerminatorGetter() {
```

Comment:
Perhaps your method name should be isTerminator?

Reply:
Yeah, I think I should change my method name using verbs. Maybe something like isTerminator or getIsTerminator.